formant_ros2_adapter/scripts/main.py

```diff
@@ -415,6 +419,16 @@ def message_callback(self, topic, base_message):
                     except Exception as e:
                         print("Error ingesting " + stream + ": " + str(e))
                         return
+                elif type(message) == BotPath:
+                    try:
+                        data = message_to_json(message)
+                        self.fclient.send_on_custom_data_channel("path_info", json.dumps(data).encode("utf-8"))
+                        self.fclient.post_json(stream, message_to_json(message))
+                    except grpc.RpcError as e:
+                        return
```

Comment:
Any reason not to print an error here?

Reply:
@ahelten No - we were just keeping it consistent with what formant had done and they never print an error there in the other statements.

Reply:
I'll go ahead and merge and we can maybe look later at whether we should add additional error logging to the module.